content/java/concepts/errors/terms/typenotpresentexception/typenotpresentexcption.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'TypeNotPresentException'
+Description: 'Occurs when an application tries to access a type using a string representing the type's name, but no definition for the type with the specified name can be found.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Errors'
+  - 'Error Handling'
+  - 'Exceptions'
+  - 'Debugging'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `TypeNotPresentException` occurs when an application tries to access a type using a string representing the type's name, but no definition for the type with the specified name can be found. This exception differs from `ClassNotFoundException` in that `ClassNotFoundException` is a checked exception, whereas this exception is unchecked. This exception can be thrown when undefined type variables are attempted to be accessed as well as when types (classes, interfaces or annotation types) are loaded. It can, in particular, be thrown by the API used to read annotations reflectively.
```

Comment:
@tarunsamanta2k20 This section is copied from https://docs.oracle.com/javase/8/docs/api/java/lang/TypeNotPresentException.html

Can you put this in your own words?

Reply:
Sure

Reply:
@tarunsamanta2k20 Thanks! This PR will be marked as invalid for now until all the plagiarism is resolved. cc: @SSwiniarski

Reply:
Let me rewrite all those stuffs